### PR TITLE
Update Readme file

### PR DESCRIPTION
**Pull request by cmcaboy**
Change the README to the Typescript NYC logo and group title

### CODE REVIEW

**Comment by cmcaboy** [comment]
@thefrankharvey Can you accept this pull request?

File: README.md [hunk 1]
```diff
@@ -1,97 +1,9 @@
-<!-- AUTO-GENERATED-CONTENT:START (STARTER) -->
 <p align="center">
```
Comment (thefrankharvey) [inline README.md]: Nice!

Comment (cmcaboy) [reply README.md]: Thank you

**Comment by cmcaboy** [comment]
Can you please merge this now?

**Pull request merged**